### PR TITLE
Switch to github-actions

### PR DESCRIPTION
**Pull request by discordier**
## Description

This changes the build flow from `travis-ci` to `github-actions`.

The change is necessary, as travis discontinued the open source minutes with the migration from https://travis-ci.org/ to https://travis-ci.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.org/MetaModels/attribute_translatedtabletext.svg)](https://travis-ci.org/MetaModels/attribute_translatedtabletext)
 [![Latest Version tagged](http://img.shields.io/github/tag/MetaModels/attribute_translatedtabletext.svg)](https://github.com/MetaModels/attribute_translatedtabletext/tags)
```
Comment (zonky2) [inline README.md:1]: add
`[Build Status](https://github.com/MetaModels/attribute_translatedtabletext/actions)`

Comment (zonky2) [reply README.md:1]: see comment

Comment (discordier) [reply README.md:2]: Fixed in 02939bc

**Review by zonky2 — APPROVED** [review]

**Review by zonky2 — APPROVED** [review]

**Review by zonky2 — APPROVED** [review]

**Comment by zonky2** [comment]
Implemented in MM 2.2

**Pull request closed without merging**